Android/GoogleDagger/.gitignore

```diff
@@ -0,0 +1,2 @@
+nugets
+nuget.config
```

Comment:
Do we need this file?

Reply:
I think it'll helpful when we use new dependency packages locally to verify the binding.